src/zenml/integrations/evidently/__init__.py

```diff
@@ -54,7 +54,7 @@ class EvidentlyIntegration(Integration):
     """[Evidently](https://github.com/evidentlyai/evidently) integration for ZenML."""
 
     NAME = EVIDENTLY
-    REQUIREMENTS = ["evidently>0.2.6"]  # supports pyyaml 6
+    REQUIREMENTS = ["evidently>0.2.6,<0.4.5"]  # supports pyyaml 6
```

Comment:
Does this fix path access issues? Why is that?

Reply:
In the Evidently PR I linked above they added `nltk.download("wordnet", quiet=True)` directly into their code - which always downloads to `~/nltk_data` for some reason. Spent quite some time debugging this and looking into how we can control the path to where NLTK would download it but couldn't get it to work and have no clue why. So decided to just ignore this for now by pinning the old Evidently version.